include/dts-functions.sh

```diff
@@ -1403,6 +1413,10 @@ main_menu_options(){
 
         if [ -n "${LOGS_SENT}" ]; then
           ${CMD_DASHARO_DEPLOY} install
+          result=$?
+          if [ "$result" -ne 0 ] && [ "$result" -ne 2 ]; then
+            send_dts_logs ask
```

Comment:
```suggestion
            send_dts_logs ask && return 0
```

So not to see `Press Enter to continue.` twice.